NPAWSampleApp/app/src/main/java/com/ooyala/sample/players/NPAWDefaultPlayerActivity.java

```diff
@@ -91,51 +70,14 @@ protected void onStop() {
     }
   }
 
-  @Override
-  protected void onRestart() {
-    super.onRestart();
-    Log.d(TAG, "Player Activity Restarted");
-    if (player != null) {
-      player.resume();
-    }
-    pluginOoyala.resumeMonitoring();
```

Comment:
i don't see you're resumingMonitoring elsewhere, and you removed this. I think the implementation is broken. Make sure to know when to start, pause, resume and stop monitoring.